lib/stink_bomb/configuration.rb

```diff
@@ -6,8 +6,31 @@ def initialize(options = {})
       self.message = options.fetch(:message) { default_message }
     end
 
+    def raise=(error_class)
+      case error_class
+      when true
+        add_bomb(RaiseBomb.new)
+      when false
```

Comment:
What if we did something like:
```ruby
def raise=
  if error_class == true
    @raise_bomb = RaiseBomb.new
  elseif ...
  else
    @raise_bomb = nil
  end
end

def bombs
  [raise_bomb + other_bomb].compact
end
```

Not the most scalable but it's nice in that each writer method handles it's own bomb.